app/routes/account._index/route.tsx

```diff
@@ -0,0 +1,15 @@
+import { LoaderFunction, redirect } from "@remix-run/node"
+import { initPortalClient } from "~/models/portal/portal.server"
+import { requireUser } from "~/utils/session.server"
+
+export const loader: LoaderFunction = async ({ request }) => {
+  const user = await requireUser(request, "/api/auth/auth0")
+  const portal = initPortalClient({ token: user.accessToken })
+
+  const getPortalUserIdResponse = await portal.getPortalUserID().catch((e) => {
```

Comment:
I think we want to be getting the users accounts here and randomly selecting one or just always taking the first. Or maybe some logic around figuring out which account they might be the owner of. But passing the userId here I dont think is doing much.

Reply:
Yeah, this was done using the `portalUserId` for now since I don't have access to the `accountId` yet
As for the logic, I think we can use two approaches:
- Always look for the owned account and select it by default 
- Select the owner at first, but when the user switches to another account save their selection and make it the default  one (using localStorage)